engine/apps/email/tasks.py

```diff
@@ -80,7 +80,7 @@ def _create_user_notification_policy_log_record(**kwargs):
     if emails_left <= 0:
         _create_user_notification_policy_log_record(
             author=user,
-            type=UserNotificationPolicyLogRecord.TYPE_PERSONAL_NOTIFICATION_FAILED,
+            type=UserNotificationPolicyLogRecord.TYPE_PERSONAL_NOTIFICATION_FINISHED,
```

Comment:
hm.. I think the FINISHED log record indicates the end of the personal policies (it stops escalation for user), so may not be the best option here (or it could be confusing). I think we could maybe add a new type here, or even consider using SUCCESS (updating the rendered log accordingly) since we did our part. Note we would have the same issue with SMS and phone calls, I think? Alternatively, yeah, ignoring FAILED log records for limit exceeded entries would work too.